CONTRIBUTING.md

```diff
@@ -0,0 +1,23 @@
+# Contribution guidelines
+
+Thank you for your interest in contributing. It is greatly appreciated.
+
+# Pull requests
+
+Code contributions to this project follow this process and set of requirements:
+
+1. As a contributor you SHOULD create an issue describing the suggested change on [the Redmine installation platform.dandigbib.org](http://platform.dandigbib.org/) in Danish. This allows the business side of the project to evaluate your suggestion. If it accepted for inclusion it will be marked for review. The only exception from this rule is if the change does not affect end users in any way. This could be documentation changes, technical tests or changes to tools.
+2. All communication on GitHub in English SHOULD be in English.
+3. The title of your pull request MUST refer to the Redmine issue number if applicable e.g. `#[issue number]: [summary of change]`. The description of your pull request SHOULD include a link to the issue.
+4. The body of the pull request SHOULD provide a short description of the suggested change and the reasoning behind the approach you have chosen.
+5. If your change affects the user interface you SHOULD include a screenshot of the result with the pull request.
+6. Your commit message MUST follow the [Drupal commit message format](https://www.drupal.org/node/52287): `Issue #[issue number] by [comma-separated usernames]: [Short summary of the change]`.
```

Comment:
Maybe we should move away from requiring the username, so the format would be: Issue #[issue number]: [Short summary of the change]